content/management/configuration-api/v2.0/index.mdx

```diff
@@ -2113,7 +2201,11 @@ curl "https://api.livechatinc.com/greetings\
 
 Returns the specified greeting.
 
-### Additional info:
+### Required scopes
+
+- `greetings_read`
```

Comment:
:O actually this requires `greetings_write`

Looks like a bug in the code but I don't know if we should fix it :|